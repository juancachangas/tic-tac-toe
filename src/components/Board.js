import React from 'react';
import FieldBox from './Field';

const fieldsArray = [0, 1, 2, 3, 4, 5, 6, 7, 8];

const Board = () => (
  <div className="Board">
    {fieldsArray.map(id => <FieldBox id={id} key={id} />)}
  </div>
);

export default Board;
