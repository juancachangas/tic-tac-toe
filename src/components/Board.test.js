import React from 'react';
import { shallow } from 'enzyme';

import Board from './Board';

describe('Board component is', () => {
  it('rendering default', () => {
    const wrapper = shallow(
      <Board />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
