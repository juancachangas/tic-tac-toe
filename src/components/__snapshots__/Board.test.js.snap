// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Board component is rendering default 1`] = `
<div
  className="Board"
>
  <Connect(Field)
    id={0}
    key="0"
  />
  <Connect(Field)
    id={1}
    key="1"
  />
  <Connect(Field)
    id={2}
    key="2"
  />
  <Connect(Field)
    id={3}
    key="3"
  />
  <Connect(Field)
    id={4}
    key="4"
  />
  <Connect(Field)
    id={5}
    key="5"
  />
  <Connect(Field)
    id={6}
    key="6"
  />
  <Connect(Field)
    id={7}
    key="7"
  />
  <Connect(Field)
    id={8}
    key="8"
  />
</div>
`;
